### PR TITLE
Particles animations added 

### PR DESCRIPTION
**Pull request by JacobKolaczkowski**
for now if anyone wants to test the three different particle animations just go into the Particle.cs class and change the Current CreateSpawnParticles() to either CreateDeathParticles() or CreateSwordBeamParticles()

### CODE REVIEW

File: Game/SpriteHandlers/SpriteFactory/ParticleSpriteFactory.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+        public static ISprite CreateSwordBeamParticles()
+        {
+            return new ParticleSprites.SwordBeamParticles(
+                TextureMap["ParticleSprites"], SpriteBatch,
+                numRows: 1,
+                numColumns: 4,
+                numberOfFrames: 4,
+                textureStartingY: 18,
```
Comment (caelummmuhhh) [inline Game/SpriteHandlers/SpriteFactory/ParticleSpriteFactory.cs:38]: Generally, I think most of our sprites are 16x16, if they're not, then they're a multiple of 8. You can make this 16 and remove the frameHeight argument.

File: Game/SpriteHandlers/SpriteFactory/ParticleSpriteFactory.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        public static ISprite CreateDeathParticles()
+        {
+
+
+            return new ParticleSprites.DeathParticles(
+                TextureMap["ParticleSprites"], SpriteBatch,
+                numRows: 1,
+                numColumns: 2,
+                numberOfFrames: 2,
+                frameHeight: 16,
```
Comment (caelummmuhhh) [inline Game/SpriteHandlers/SpriteFactory/ParticleSpriteFactory.cs:24]: frameWidth and frameHeight already have default values of 16. I don't think this is necessary. This applies to the rest of the factory too.

File: Game/Particles/Particle.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    public class Particle : IParticle
+    {
+        public ISprite Part;
+        private readonly Game1 game;
+        public Particle(Game1 game)
+        {
+          
+           
+            Part = SpriteFactory.CreateSpawnParticles();
+            this.game = game;
```
Comment (caelummmuhhh) [inline Game/Particles/Particle.cs:23]: I don't think Game1 is a necessary parameter for this class. I see you used it to determine the position. In the future, particles spawn position will actually be based on enemies' position instead. You could have a position be passed instead.

**Review by caelummmuhhh — DISMISSED** [review]
Looks good for the most part. See comments I left. Also, if you could make the white spacing more consistent, that would be great.

**Review by Landon-Nagle — APPROVED** [review]

**Comment by Landon-Nagle** [comment]
I'm just gonna resolve the changes and merge in an effort make sure we can turn in on time.

**Pull request merged**